ts/Core/Pointer.ts

```diff
@@ -404,6 +460,33 @@ class Pointer {
         chart.mouseDownY = this.mouseDownY = e.chartY;
     }
 
+    /**
+     * Get selection box to calculate extremes
+     * @private
+     * @function Highcharts.Pointer#getSelectionBox
+     * @emits afterGetSelectionBox
+     */
+    public getSelectionBox(marker: SVGElement): Partial<BBoxObject> {
+        let x = marker.attr ? +marker.attr('x') : marker.x,
+            y = marker.attr ? +marker.attr('y') : marker.y,
+            width = marker.attr ?
+                marker.attr('width') :
+                marker.width,
+            height = marker.attr ?
+                marker.attr('height') :
+                marker.height;
+
+        let event = { x, y, width, height, marker };
+
+        fireEvent(
+            this,
+            'afterGetSelectionBox',
```

Comment:
As above, unless we're extending on the default functionality, don't use the `after*` pattern.